src/ir/mod.rs

```diff
@@ -526,6 +555,12 @@ pub struct Normalize {
     pub ty: Ty,
 }
 
+#[derive(Clone, PartialEq, Eq, Hash, PartialOrd, Ord)]
```

Comment:
This feels like a nice place for a comment that explains the difference between *normalizing* and *unselected normalize* and perhaps gives a brief example (of Rust code).